### PR TITLE
Tpd 980 - New Involvement Creation dialog box

### PR DESCRIPTION
**Pull request by jrred**
This PR is just for the dialog box shown in the Jira card for TPD-980. Since it requires affiliation with a model, the PR also includes a new model, InvolvementModel.cs.

Jira card: https://touchpointsoftware.atlassian.net/secure/RapidBoard.jspa?rapidView=5&projectKey=TPD&modal=detail&selectedIssue=TPD-980&assignee=5e9da7220d58350c2be09a8f&assignee=557058%3A6bf2cb0a-9f56-424e-a17a-b2a61a5f4cb0

[touchpointsoftware.atlassian.net/browse/TPD-980](https://touchpointsoftware.atlassian.net/browse/TPD-980)

### CODE REVIEW

**Comment by jrred** [comment]
This has build issues - disregard for now.

File: CmsWeb/CMSWeb.csproj [hunk 1]
```diff
@@ -1246,6 +1247,7 @@
     <Content Include="Content\font-awesome-4.0.3\fonts\fontawesome-webfont.svg" />
     <Content Include="Content\images\check.png" />
     <Content Include="Content\img\textarealine.png" />
+    <Content Include="Content\Site.css" />
```
Comment (brondavies) [inline CmsWeb/CMSWeb.csproj:1250]: This was probably created by the scaffolding tool - we don't need it though.

File: CmsWeb/Content/Site.css [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    padding-right: 15px;
+}
+
+/* Set width on the form input elements since they're 100% wide by default */
+input,
+select,
+textarea {
+    max-width: 280px;
+}
+
```
Comment (brondavies) [inline CmsWeb/Content/Site.css:18]: Delete this file

File: CmsWeb/Areas/Dialog/Views/AddInvolvement/index.cshtml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+                <h4 class="modal-title">Add New Involvement</h4>
+            </div>
+            <div class="modal-body">
+                <div class="row">
+                    <div class="col-sm-6">
+                        <div class="form-group">
+                            <label for="inv_InvolvementName" class="control-label">Name</label>
+                            @Html.TextBox("inv.InvolvementName", Model.InvolvementName, new { @class = "form-control" })
```
Comment (brondavies) [inline CmsWeb/Areas/Dialog/Views/AddInvolvement/index.cshtml:15]: There's a build error here. See https://builds.tpsdb.com/viewLog.html?buildTypeId=Bvcms_Build&buildId=14824&tab=buildLog&logTab=tree&filter=debug&expand=all&_focus=6894

**Pull request closed without merging**